bundles/org.openhab.binding.automower/src/main/resources/ESH-INF/i18n/automower_de.properties

```diff
@@ -0,0 +1,28 @@
+binding.automower.name = Automower Binding
+binding.automower.description = Das Automower Binding erlaubt die Interaktion mit Husqvarna Automower M�hrobotern mit Automower Connect Unterst�tzung.
+
+thing-type.automower.bridge.label = Automower Connect Bridge
+thing-type.automower.bridge.description = Erlaubt die Kommunikation mit der Husqvarna Automower Connect API
+
+thing-type.config.automower.bridge.appKey.label = Application Key
+thing-type.config.automower.bridge.appKey.description = Der Application Key wird f�r die Kommunication mit der Automower Connect API ben�tigt. Um diesen zu erhalten muss eine Anwendung auf der Husqvarna Website registriert werden. Diese Anwendung muss mit der "Authentication API" und der "Automower Connect API" verkn�pft werden.
+
+thing-type.config.automower.bridge.userName.label = Benutzername
+thing-type.config.automower.bridge.userName.description = Der Benutzername f�r den der Application Key ausgestellt wurde.
+
+thing-type.config.automower.bridge.password.label = Passwort
+thing-type.config.automower.bridge.password.description = Das Passwort f�r den angegebenen Benutzer.
+
+thing-type.config.automower.bridge.pollingInterval.label = Polling Intervall
+thing-type.config.automower.bridge.pollingInterval.description = Das Intervall in dem die Verbindung mit dem Automower Connect API �berpr�ft werden soll
+
+
+thing-type.automower.automower.label = Automower
+thing-type.automower.automower.description = Ein Husqvarna Automower
+
+thing-type.config.automower.automower.mowerId.label = Automower Id
+thing-type.config.automower.automower.mowerId.label = Die interne ID des Automowers die von der Automower Connect API benutzt wird um einen Automower zu identifizieren
+
+thing-type.config.automower.automower.pollingInterval.label = Polling Intervall
+thing-type.config.automower.automower.pollingInterval.label = Das Intervall in dem der Status des Automowers abgefragt werden soll (in Sekunden). Der Standardwert ist 300s (5min) 
```

Comment:
The unit is defined via the XML file and displayed separately in the UI. So, it can be removed here.

Reply:
ok